core/db/migrate/20151002023248_add_foreign_key_to_shipping_method_stock_location.rb

```diff
@@ -0,0 +1,6 @@
+class AddForeignKeyToShippingMethodStockLocation < ActiveRecord::Migration
+  def change
+    add_foreign_key :spree_shipping_method_stock_locations, :spree_shipping_methods, column: :shipping_method_id
+    add_foreign_key :spree_shipping_method_stock_locations, :spree_stock_locations, column: :stock_location_id
```

Comment:
Oh man... Can't wait till rails 5 is out.

``ruby
t.belongs_to :shipping_method, foreign_key: true
```